bedrock/bedrock-common/src/main/java/com/nukkitx/protocol/bedrock/packet/CreativeContentPacket.java

```diff
@@ -2,20 +2,26 @@
 
 import com.nukkitx.protocol.bedrock.BedrockPacket;
 import com.nukkitx.protocol.bedrock.BedrockPacketType;
-import com.nukkitx.protocol.bedrock.data.inventory.ItemData;
+import com.nukkitx.protocol.bedrock.data.inventory.ItemDataInstance;
 import com.nukkitx.protocol.bedrock.handler.BedrockPacketHandler;
 import it.unimi.dsi.fastutil.ints.Int2ObjectMap;
 import it.unimi.dsi.fastutil.ints.Int2ObjectOpenHashMap;
 import lombok.Data;
 import lombok.EqualsAndHashCode;
 
+
+/**
+ * CreativeContent is a packet sent by the server to set the creative inventory's content for a player.
+ * Introduced in 1.16, this packet replaces the previous method - sending an InventoryContent packet with
+ * creative inventory window ID.
+ */
 @Data
 @EqualsAndHashCode(doNotUseGetters = true, callSuper = false)
 public class CreativeContentPacket extends BedrockPacket {
     /**
      * Item entries for the creative menu. Each item must have a unique ID for the net ID manager
      */
-    private final Int2ObjectMap<ItemData> entries = new Int2ObjectOpenHashMap<>();
+    private final Int2ObjectMap<ItemDataInstance> entries = new Int2ObjectOpenHashMap<>();
```

Comment:
I'm confused why this was changed. The network ID is the map key so the extra class seems a bit pointless.

Reply:
I wasn't sure on that one, but looking at other packets (like SetSlotPacket) they all have networkID's as well so it seemed an idea to move it to an instance. However if you prefer to flatten that structure into the packets then that can be changed back.

Reply:
Is the network id not included in the ItemData class though?

Reply:
I don't think so as its a runtime ID held by the server for that specific instance of itemstack and some itemstacks may not have it. I think needs to be separate either as an index on the collection like SupremeMortal has or a separate object. 
I've flattened the structure and worked how to provide backwards compatibility in the InventoryContentsPacket (my main reason for the change) so this should be resolved now.